packages/web-app-admin-settings/src/components/General/AppearanceSettings.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <h2 class="oc-py-s" v-text="$gettext('Apperance')" />
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, unref } from 'vue'
+import { useStore } from 'web-pkg/src/composables'
+
+export default defineComponent({
+  name: 'Appearance Settings',
+  setup() {
+    const store = useStore()
+    const currentTheme = computed(() => store.getters.configuration.currentTheme)
+    console.log(unref(currentTheme))
+  }
+})
+</script>
```

Comment:
🙈

Reply:
Would be best if you could move the panel and its usage into a separate branch for your next issue and exclude it from this PR :-)